tsl/src/nodes/decompress_chunk/compressed_batch.c

```diff
@@ -265,6 +265,12 @@ decompress_column(DecompressContext *dcontext, DecompressBatchState *batch_state
 		column_values->buffers[1] = arrow->buffers[1];
 		column_values->buffers[2] = NULL;
 		column_values->buffers[3] = NULL;
+
+		if (value_bytes == 1 && column_description->typid == BOOLOID)
+		{
+			/* Overrode the decompression type for bool columns */
```

Comment:
```suggestion
		if (column_description->typid == BOOLOID)
		{
			/* The bool columns have a dedicated storage format. */
```

The condition for `value_bytes` looks unneeded, we don't work differently with bools of any other sizes.

Reply:
Yeah, BOOLOID implies value_bytes == 1